web/src/components/atomic/Icon.vue

```diff
@@ -66,7 +66,7 @@
     />
     <path
       fill="currentColor"
-      d="M12,4a8,8,0,0,1,7.89,6.7A1.53,1.53,0,0,0,21.38,12h0a1.5,1.5,0,0,0,1.48-1.75,11,11,0,0,0-21.72,0A1.5,1.5,0,0,0,2.62,12h0a1.53,1.53,0,0,0,1.49-1.3A8,8,0,0,1,12,4Z"
+      d="M12,4a8,8,0,0,1,7.89,6.7A1.33,1.33,0,0,0,21.38,12h0a1.3,1.3,0,0,0,1.48-1.75,11,11,0,0,0-21.72,0A1.3,1.3,0,0,0,2.62,12h0a1.33,1.33,0,0,0,1.49-1.3A8,8,0,0,1,12,4Z"
```

Comment:
Why was this changed?

Reply:
Can't remember I did this. Can revert it.